src/styles/theme/defaultTheme.ts

```diff
@@ -0,0 +1,75 @@
+import {createTheme} from "@mui/material";
+
+export const defaultTheme = createTheme({
+    palette: {
+        mode: "light",
+        primary: {
+            main: "#2D3E6B",
+            contrastText: "#FFFFFF",
+            dark: "#243664",
```

Comment:
dark and light are not better name choice. What if new theme will be vise-versa

Reply:
these are the names of the Material UI theme, I just overwritten them with our colors https://mui.com/material-ui/customization/default-theme/